main.js

```diff
@@ -441,12 +481,59 @@ function reset() {
 }
 
 function winner() {
-    if (frogY == 50) {
-        currentScore = score + 300;
-
-        console.log("Congratu-freaking-lations!");
-        reset();
 
+      //if frog jumps in first homespace and it is empty
+      if (((frogY == 50) && (frogX > 5 && frogX < 25)) && homeSpaceArray[0] == 0) {
```

Comment:
All this code can be done with a for loop and use the variable 'i' inside homeSpaceArray[i] == 0

Reply:
I wasn't sure how to do that since it was dependent on the frogX position being within a certain number for every home space, the first one being between 5 and 25, the second between 95 and 115, etc.